headapps/MvpSite/MvpSite.Rendering/ViewComponents/Apply/ApplicationFormViewComponent.cs

```diff
@@ -432,6 +432,14 @@ private async Task ExecuteContributionsStep(ApplicationFormModel model)
             && model.ContributionDate.Value >= model.CurrentSelection?.ApplicationsEnd.AddMonths(-_options.TimeFrameMonths)
             && model.ContributionDate.Value <= model.CurrentSelection.ApplicationsEnd)
         {
+            if (model.CurrentApplication?.Contributions.Count >= Convert.ToInt32(model.MaxContributions?.Value))
+            {
+                model.NextStep = ApplicationStep.Contributions;
+                await LoadProducts(model);
+                ModelState.AddModelError(string.Empty, model.ContributionMinLimitMessage?.Value ?? "You have reached maximum number of contributions");
```

Comment:
This should be outputting the ContributionsMaxLimit value not the MinLimit value right?